identity/src/main/java/com/stripe/android/identity/navigation/IdentityTopLevelDestination.kt

```diff
@@ -0,0 +1,155 @@
+package com.stripe.android.identity.navigation
+
+import androidx.navigation.NamedNavArgument
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavController
+import com.stripe.android.identity.analytics.IdentityAnalyticsRequestFactory
+import com.stripe.android.identity.networking.models.Requirement
+
+internal abstract class IdentityTopLevelDestination(
```

Comment:
Note: The content of this class is not changed but just moved from `NavigationDestinations.kt`